workflow-bot-app/src/test/java/com/symphony/bdk/workflow/engine/WorkflowDirectGraphBuilderTest.java

```diff
@@ -32,7 +32,7 @@ void buildWorkflowDirectGraph_approvalFlow() throws Exception {
     Workflow workflow = SwadlParser.fromYaml(getClass().getResourceAsStream("/graph/approval.swadl.yaml"));
     workflowDirectGraphBuilder = new WorkflowDirectGraphBuilder(workflow, eventMapper);
     WorkflowDirectGraph directGraph = workflowDirectGraphBuilder.build();
-    assertThat(directGraph.getDictionary()).hasSize(8);
+    assertThat(directGraph.getDictionary()).hasSize(9);
```

Comment:
what this change is related to?

Reply:
the default timeout node for FormRepliedEvent, which was not created in graph builder before when using subprocess.